docs/lms_config.ini

```diff
@@ -7,7 +7,7 @@ version = latest
 
 # below are the pdf settings for the pdf file
 [pdf_settings]
-pdf_base = https://media.readthedocs.io/pdf/open-edx-learner-guide
+pdf_base = https://media.readthedocs.org/pdf/open-edx-learner-guide
```

Comment:
You can't access https://media.readthedocs.org/pdf/open-edx-learner-guide itself-- it's just the base for the URL.

https://media.readthedocs.org/pdf/open-edx-learner-guide/latest/open-edx-learner-guide.pdf (which is what is ultimately constructed) does work.

I'll leave the PDF stuff in for now as I don't want to spend more time on this hackathonish project.